conf/test.config

```diff
@@ -20,6 +20,6 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input  = params.pipelines_testdata_base_path + 'viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
```

Comment:
what's wrong with using the parameter?

Reply:
It doesn't work in Platform

Reply:
🤯 wasn't that one of the reasons why we changed to that format, @maxulysse ?